proprietary/tokens/src/brand/ams/brand.color.tokens.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
This file is temporary – it’s useful for refactoring. I will remove it before publishing the PR.